app/views/two_factor_authentication/otp_expired/show.html.erb

```diff
@@ -31,5 +31,13 @@
           text: t('links.contact_support', app_name: APP_NAME),
           new_tab: true,
         },
-      ],
+        {
+          url: @authentication_options_path,
+          text: t('two_factor_authentication.login_options_link_text'),
+        },
```

Comment:
Related to my last comment, I wonder what should be shown when the user is adding a phone to their existing account, since "Choose another authentication method" doesn't really make sense at all in that context.

Reply:
I am going to check in with UX on this because this is the feeling that I am getting. I will put results here as well.

Reply:
@aduth Final decision on this is to allow users to choose another auth method when adding a phone to an existing account. The rationale from UX is that we should give all of the options possible when it comes to choosing an auth method through that pathway

Reply:
I'm not sure I understand what that will mean in practice, since the `/authentication_methods_setup` route isn't available for accounts which already exist, and might make assumptions about being a setup flow for new accounts.